packages/core/src/i18n/translations.ts

```diff
@@ -13,164 +13,163 @@
 
 export const dict = {
   de: {
-    'Sign In': 'Anmelden',
-    'Sign Up': 'Registrieren',
-    'Sign Out': 'Abmelden',
-    'Sign in to your account': 'Melden Sie sich mit Ihrem Account an',
-    Username: 'Benutzername',
```

Comment:
How'd you do this?  Did we want to leverage https://www.npmjs.com/package/prettier-plugin-sort-json to do this automatically?

Reply:
I just used a VS code extension. Sort JS Objects or something. Super simple.